### PR TITLE
G0 5372 new guideline

### PR DESCRIPTION
**Pull request by shalinee-walkover**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Other information**:

### CODE REVIEW

File: apps/web-giddh/src/app/home/components/bank-accounts/bank-accounts.component.scss [hunk 1]
```diff
@@ -1,3 +1,4 @@
+@import "../../../../assets/css/color.scss";
 p.bankName {
```
Comment (sadik-walkover) [inline apps/web-giddh/src/app/home/components/bank-accounts/bank-accounts.component.scss:2]: p.bankName class name should be .bank-name

File: apps/web-giddh/src/app/daybook/daybook.component.scss [hunk 2]
```diff
@@ -46,7 +47,7 @@
     align-items: center;
     font-size: 18px;
     margin-left: 7px;
-    border: 1px solid #d9d9d9;
+    border: 1px solid $gainsboro;
     border-radius: 2px;
     color: #1a237e;
```
Comment (sadik-walkover) [inline apps/web-giddh/src/app/daybook/daybook.component.scss:52]: use color var

File: apps/web-giddh/src/app/home/components/bank-accounts/bank-accounts.component.scss [hunk 3]
```diff
@@ -1,3 +1,4 @@
...
     font-size: 14px;
     color: #1a237e;
```
Comment (sadik-walkover) [inline apps/web-giddh/src/app/home/components/bank-accounts/bank-accounts.component.scss:4]: use color var

File: apps/web-giddh/src/app/home/components/cr-dr-list/cr-dr-list.component.scss [hunk 4]
```diff
@@ -1,3 +1,4 @@
+@import "../../../../assets/css/color.scss";
 p.bankName {
     font-size: 14px;
     color: #1a237e;
```
Comment (sadik-walkover) [inline apps/web-giddh/src/app/home/components/cr-dr-list/cr-dr-list.component.scss:4]: user color var

File: apps/web-giddh/src/app/home/components/revenue/revenue-chart.component.scss [hunk 5]
```diff
@@ -1,3 +1,4 @@
+@import "../../../../assets/css/color.scss";
 .secondryDarkbg span {
     background: #087e7d;
```
Comment (sadik-walkover) [inline apps/web-giddh/src/app/home/components/revenue/revenue-chart.component.scss:3]: user color var

File: apps/web-giddh/src/app/home/components/revenue/revenue-chart.component.scss [hunk 6]
```diff
@@ -122,7 +123,7 @@ a.connectBnk {
     display: inline-flex;
     justify-content: center;
     align-items: center;
-    border-right: 1px solid #d1d1d1;
+    border-right: 1px solid $gainsboro;
     color: #262626;
```
Comment (sadik-walkover) [inline apps/web-giddh/src/app/home/components/revenue/revenue-chart.component.scss:127]: use color var

File: apps/web-giddh/src/app/import-excel/import-excel.component.scss [hunk 7]
```diff
@@ -1,3 +1,4 @@
+@import "../../assets/css/color.scss";
 .steps_wizard {
     padding: 30px 0;
     background: #fff;
```
Comment (sadik-walkover) [inline apps/web-giddh/src/app/import-excel/import-excel.component.scss:4]: use var

File: apps/web-giddh/src/app/import-excel/import-report/import-report.component.html [hunk 8]
```diff
@@ -5,22 +5,22 @@
...
 
-            <div class="row" style="margin: 0">
+            <div>
 
                 <div class="col-xs-6 mb-2" *ngFor="let item of importStatusResponse.results">
 
-                    <div class="r-card">
+                    <div class="import-card">
 
                         <i class="hover-orange" style="position: absolute;right: 15px;cursor: pointer;z-index: 2;"
```
Comment (sadik-walkover) [inline apps/web-giddh/src/app/import-excel/import-report/import-report.component.html:20]: remove inline css

File: apps/web-giddh/src/app/import-excel/import-report/import-report.component.html [hunk 9]
```diff
@@ -5,22 +5,22 @@
...
                             (click)="downloadItem(item)">
                             <svg xmlns="http://www.w3.org/2000/svg" width="13.333" height="15" viewBox="0 0 13.333 15"
-                                class="svg_hover_orange">
+                                class="download-import">
                                 <path class=""
```
Comment (sadik-walkover) [inline apps/web-giddh/src/app/import-excel/import-report/import-report.component.html:24]: remove blank css

File: apps/web-giddh/src/app/import-excel/import-report/import-report.component.scss [hunk 10]
```diff
@@ -1,15 +1,18 @@
...
 
-    label.text-gray {
-        color: #646464;
+.import-data-tble{
+    height: 68vh; 
+    overflow-y: auto;
+    .import-card {
+        padding: 15px;
+        background-color: white;
+        box-shadow: 0px 3px 7px #00000026;
```
Comment (sadik-walkover) [inline apps/web-giddh/src/app/import-excel/import-report/import-report.component.scss:9]: my suggestion we can use color var

File: apps/web-giddh/src/app/import-excel/import-type-select/import-type-select.component.html [hunk 11]
```diff
@@ -1,5 +1,5 @@
 <div class="col-xs-12">
-    <div class="pd2 mrT2">
+    <div class="pt-2 mrT2">
```
Comment (sadik-walkover) [inline apps/web-giddh/src/app/import-excel/import-type-select/import-type-select.component.html:2]: capital class name mrT2

File: apps/web-giddh/src/app/import-excel/import-report/import-report.component.html (hunk 8 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/import-excel/import-report/import-report.component.html:20]: I am not worked on this line

File: apps/web-giddh/src/app/home/components/bank-accounts/bank-accounts.component.scss (hunk 1 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/home/components/bank-accounts/bank-accounts.component.scss]: fixed

File: apps/web-giddh/src/app/daybook/daybook.component.scss (hunk 2 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/daybook/daybook.component.scss]: fixed

File: apps/web-giddh/src/app/home/components/bank-accounts/bank-accounts.component.scss (hunk 3 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/home/components/bank-accounts/bank-accounts.component.scss:4]: fixed

File: apps/web-giddh/src/app/home/components/cr-dr-list/cr-dr-list.component.scss (hunk 4 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/home/components/cr-dr-list/cr-dr-list.component.scss]: fixed

File: apps/web-giddh/src/app/home/components/revenue/revenue-chart.component.scss (hunk 5 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/home/components/revenue/revenue-chart.component.scss]: fixed

File: apps/web-giddh/src/app/home/components/revenue/revenue-chart.component.scss (hunk 6 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/home/components/revenue/revenue-chart.component.scss]: fixed

File: apps/web-giddh/src/app/import-excel/import-excel.component.scss (hunk 7 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/import-excel/import-excel.component.scss]: fixed

File: apps/web-giddh/src/app/import-excel/import-report/import-report.component.html (hunk 9 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/import-excel/import-report/import-report.component.html]: fixed

File: apps/web-giddh/src/app/import-excel/import-report/import-report.component.scss (hunk 10 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/import-excel/import-report/import-report.component.scss]: fixed

File: apps/web-giddh/src/app/import-excel/import-type-select/import-type-select.component.html (hunk 11 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/import-excel/import-type-select/import-type-select.component.html]: fixed

**Pull request merged**